src/crystal/system/win32/time.cr

```diff
@@ -69,15 +69,15 @@ module Crystal::System::Time
 
   def self.load_localtime : ::Time::Location?
     if LibC.GetTimeZoneInformation(out info) != LibC::TIME_ZONE_ID_UNKNOWN
-      initialize_location_from_TZI(info)
+      initialize_location_from_tzi(info)
     end
   end
 
   def self.zone_sources : Enumerable(String)
     [] of String
   end
 
-  private def self.initialize_location_from_TZI(info)
+  private def self.initialize_location_from_tzi(info)
```

Comment:
I think this might be overshooting. TZI is an acronym ("Time Zone Information") and IMO it makes sense to use caps for that.
In contrast to the performance improving refactoring of `Enumerable` methods, this is a purely stylistic change. And there is no established style guide decision on this. Can you please remove this change?
I'm open to adopting whatever style makes sense, but that needs a dedicated discussion and manifestation in the style guide.

Reply:
Acronyms are usually lower cased in Crystal, but I can remove it of course.

Reply:
There are many more acronyms found in the Crystal's source code, but none of them is used in the capitalized form. This is the only exception I've found, also a private method. I'd prefer to not leave exceptions like this.

Reply:
What's the status here? Should I revert it this exception?

Reply:
@Sija  revert so this is easier to land

Reply:
done!